rio-orphans/rio-orphans.cabal

```diff
@@ -7,7 +7,7 @@ cabal-version: 1.12
 -- hash: dd6b35b998c98b016949383b90c84c7de4983d5f2b68acaf535908f0c342ad6c
 
 name:           rio-orphans
-version:        0.1.1.0
+version:        0.1.2.0
```

Comment:
This requires an update to the `package.yaml` file https://github.com/commercialhaskell/rio/blob/856ed953f13442f87269feaedf5e92d184dc07f7/rio-orphans/package.yaml#L2

Reply:
Thanks for pointing that out.